test/ActiveLogin.Identity.Swedish.AspNetCore.Test/Validation/SwedishCoordinationNumberAttribute_IsValid.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using ActiveLogin.Identity.Swedish.AspNetCore.Validation;
+using Xunit;
+
+namespace ActiveLogin.Identity.Swedish.AspNetCore.Test.Validation
+{
+    /// <remarks>
+    /// Tested with official test Coordination Numbers from Skatteverket:
+    /// https://skatteverket.entryscape.net/store/9/resource/1887
+    /// </remarks>
+    public class SwedishCoordinationNumberAttribute_IsValid
+    {
+        [Fact]
+        public void Returns_Valid_When_Valid_Coordination_Number()
+        {
+            var model = new SampleValidationModel("230161-2392");
```

Comment:
Swap this with a number from the approved list of testdata, see for example:
https://swedish.identityinfo.net/coordinationnumber/testdata

Reply:
Done, I updated the source link as well.